grpc/testing/messages.proto

```diff
@@ -277,6 +296,10 @@ message ClientConfigureRequest {
 // Response for updating a test client's configuration.
 message ClientConfigureResponse {}
 
+message MemorySize {
```

Comment:
This change is unrelated to the whole metadata support feature. Looks like a year old change that needed to be merged here :-)